manager/test/api/src/test/resources/test-plan-data/searching/queries/misc/001_query-api-catalog.resttest

```diff
@@ -1,41 +1,41 @@
-POST /search/apiCatalog/entries admin/admin

```

Comment:
I think here is the line ending wrong?

Reply:
Yah, I think that came from Bastian's original commit. I'll back it out and push over the top of this commit unless someone tells me there's something we should have preserved in here.